backend/apps/core/api/csrf_token.py

```
@@ -0,0 +1,7 @@
+from django.http import JsonResponse
```

Ruff (0.8.2)

Missing docstring in public module

(D100)